site/src/sphinx/client-retry.rst

```diff
@@ -335,6 +343,37 @@ This will produce only single request and response log pair regardless how many
 
     Please refer to :ref:`nested-log`, if you are curious about how this works internally.
 
+``RetryingClient`` with circuit breaker
+---------------------------------------
+
+You might want to use :ref:`client-circuit-breaker` with :api:`RetryingHttpClient` using decorator_:
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.circuitbreaker.CircuitBreakerStrategy;
+    import com.linecorp.armeria.client.circuitbreaker.CircuitBreakerHttpClientBuilder;
+
+    CircuitBreakerStrategy cbStrategy = CircuitBreakerStrategy.onServerErrorStatus();
+    RetryStrategy myRetryStrategy = new RetryStrategy(){ ... };
+
+    HttpClient client = new HttpClientBuilder(...)
+            .decorator(new CircuitBreakerHttpClientBuilder(cbStrategy).newDecorator())
+            .decorator(new RetryingHttpClientBuilder(myRetryStrategy).newDecorator())
+            .build();
+
+    final AggregatedHttpMessage res = client.execute(...).aggregate().join();
+
+This decorates :api:`CircuitBreakerHttpClient` with :api:`RetryingHttpClient` so that the :api:`CircuitBreaker`
+judges every request and retried request as successful or failed. If the failure rate exceeds a certain
+threshold, it raises a :api:`FailFastException`.
+
+You have to be very cautious implementing your own :api:`RetryStrategy` in this situation.
```

Comment:
How about adding some example to make the explanation easier to understand? For example:

```
When using both :api:`CircuitBreakerHttpClient` and :api:`RetryingHttpClient`, you need to 
write a custom :api:`RetryStrategy` so that the :api:`RetryingHttpClient` does not attempt
a retry unnecessarily when the circuit is open, e.g.

<example that stops retrying on FailFastException>

.. note::

    You may want to allow retrying even on :api:`FailFastException` when your endpoint is
    configured with `client-side load balancing`_ because the next attempt might be sent to
    the next available endpoint.
```

Not that the example above is the ground truth, though. Please feel free to improve it as you want.

Reply:
There's nothing to improve further. Thank you~!